src/library/Uno.Material/Styles/Controls/ToggleSwitch.xaml

```diff
@@ -5,7 +5,7 @@
 					xmlns:android="http:///umbrella/ui/android"
 					xmlns:xamarin="http:///umbrella/ui/xamarin"
 					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-					xmlns:wasm="http://uno.ui/wasm"
+					xmlns:wasm="http://umbrella/ui/wasm"
```

Comment:
@NicolasChampagne do you have a clear idea why it's not working with uno the namespace ?

Reply:
Nothing definitive, but its clearly only not working in the library files.

Reply:
@NicolasChampagne can you maybe enter a new issue so it can be investigated even if we have a workaround for now ;) Thanks in advance !